### ISSUE TITLE
Wrong event type in EventEmitterBuilders

### ISSUE THREAD

**Issue by PerWiklander** [OutWatch/outwatch#19]
The simple version of `apply` in `MouseEventEmitterBuilder` looks like it has the wrong event type (`InputEvent`) in:

```scala
def apply[T](t: T) =
    GenericMappedEmitterBuilder(EventEmitter(eventType, _:Observer[InputEvent]), (_: Event) => t)
```

See: https://github.com/OutWatch/outwatch/blob/master/src/main/scala/outwatch/dom/helpers/EmitterBuilder.scala#L63

The same goes for the other EventEmitterBuilders.

**Comment by LukaJCB** [comment]
Hmm, I think it actually doesn't matter which Event type is used for the currying, but I agree it doesn't look as nice, probably should just change it to an `Event` for all of the Builders. 

Or did you find something where this results in some unexpected behaviour, or even breaks something?

**Comment by PerWiklander** [comment]
Nope, haven't found any errors, just thought it looked strange, so the base Event sounds good.

 /Per

**Comment by LukaJCB** [comment]
Fixed in 77242f2706139be5a4e79f76ee80dd6c7ae6b397 :) Thanks for your help!
